src/telemetryReporter.ts

```diff
@@ -196,10 +196,10 @@ export default class TelemetryReporter<V extends string = string, A extends stri
 	 * @param extensionVersion The version of the extension sending the event
 	 * @param key The AI Key to use
 	 */
-	constructor(extensionId: string, extensionVersion: string, key: string) {
+	constructor(extensionId: string, _extensionVersion: string, key: string) {
 		// Try to initialize the reporter, but don't throw if it fails so we don't break the extension
 		try {
-			this._telemetryReporter = new VsCodeTelemetryReporter(extensionId, extensionVersion, key);
+			this._telemetryReporter = new VsCodeTelemetryReporter(key);
```

Comment:
Did you verify that the events are still being sent up with the same name? Previously the extension ID was used to prefix the events, so is that still happening (and just retrieved through some other way?)

Reply:
And extensionVersion was included as a property in the events as well

Reply:
These properties are _supposed_ to be getting read automatically from the extension's `package.json` with the new version of `@vscode/extension-telemetry`, but that appears to not be happening:

![image](https://github.com/user-attachments/assets/89fe3253-9449-452e-b7a8-4d70cc563dde)

Debugging why not, but that would just lead to another issue: the package name is `mssql`, but we use the old version to manually override to `vscode-mssql` in that extension to avoid conflicts with ADS's built-in `mssql` extension.  That seems like something we can work around, though; it'll just take a combination of political will and an engineering work-around.